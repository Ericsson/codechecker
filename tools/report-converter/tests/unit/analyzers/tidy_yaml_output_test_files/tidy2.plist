<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>diagnostics</key>
	<array>
		<dict>
			<key>category</key>
			<string>clang</string>
			<key>check_name</key>
			<string>clang-diagnostic-unused-variable</string>
			<key>description</key>
			<string>unused variable 'y'</string>
			<key>issue_hash_content_of_line_in_context</key>
			<string>9f5469099db1a2765912c8254b32e90b</string>
			<key>location</key>
			<dict>
				<key>col</key>
				<integer>7</integer>
				<key>file</key>
				<integer>0</integer>
				<key>line</key>
				<integer>5</integer>
			</dict>
			<key>path</key>
			<array>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>7</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>5</integer>
					</dict>
					<key>message</key>
					<string>unused variable 'y'</string>
				</dict>
			</array>
			<key>type</key>
			<string>clang-tidy-yaml</string>
		</dict>
		<dict>
			<key>category</key>
			<string>clang</string>
			<key>check_name</key>
			<string>clang-diagnostic-division-by-zero</string>
			<key>description</key>
			<string>remainder by zero is undefined</string>
			<key>issue_hash_content_of_line_in_context</key>
			<string>0758808c01c22b4a346799d1156c13a5</string>
			<key>location</key>
			<dict>
				<key>col</key>
				<integer>12</integer>
				<key>file</key>
				<integer>0</integer>
				<key>line</key>
				<integer>13</integer>
			</dict>
			<key>path</key>
			<array>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>12</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>13</integer>
					</dict>
					<key>message</key>
					<string>remainder by zero is undefined</string>
				</dict>
			</array>
			<key>type</key>
			<string>clang-tidy-yaml</string>
		</dict>
	</array>
	<key>files</key>
	<array>
		<string>files/test2.cpp</string>
	</array>
	<key>metadata</key>
	<dict>
		<key>analyzer</key>
		<dict>
			<key>name</key>
			<string>clang-tidy-yaml</string>
		</dict>
		<key>generated_by</key>
		<dict>
			<key>name</key>
			<string>report-converter</string>
			<key>version</key>
			<string>x.y.z</string>
		</dict>
	</dict>
</dict>
</plist>
