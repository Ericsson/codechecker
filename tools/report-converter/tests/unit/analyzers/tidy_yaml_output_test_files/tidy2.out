---
MainSourceFile:  'files/test2.cpp'
Diagnostics:
  - DiagnosticName:  clang-diagnostic-unused-variable
    DiagnosticMessage:
      Message:         'unused variable ''y'''
      FilePath:        'files/test2.cpp'
      FileOffset:      49
      Replacements:    []
      Ranges:
        - FilePath:        'files/test2.cpp'
          FileOffset:      49
          Length:          1
    Level:           Warning
    BuildDirectory:  'files'
  - DiagnosticName:  clang-diagnostic-division-by-zero
    DiagnosticMessage:
      Message:         remainder by zero is undefined
      FilePath:        'files/test2.cpp'
      FileOffset:      122
      Replacements:    []
      Ranges:
        - FilePath:        'files/test2.cpp'
          FileOffset:      124
          Length:          1
    Level:           Warning
    BuildDirectory:  'files'
...
